packages/oqamldebug/oqamldebug.0.9.5/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "sebastien.fricker@gmail.com"
+build: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  ["%{make}%"]
+  ["%{make}%" "install" ]
+]
+remove: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  [ "make" "uninstall" ]
```

Comment:
On 03/07/2015 23:54, David Sheets wrote:
>
> In packages/oqamldebug/oqamldebug.0.9.5/opam
> <https://github.com/ocaml/opam-repository/pull/4366#discussion_r33881672>:
>
> > @@ -0,0 +1,12 @@
> > +opam-version: "1"
> > +maintainer: "sebastien.fricker@gmail.com"
> > +build: [
> > +  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
> > +  ["%{make}%"]
> > +  ["%{make}%" "install" ]
> > +]
> > +remove: [
> > +  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
> > +  [ "make" "uninstall" ]
>
> I think this needs |make| rather than |"make"| in order to work with
> systems where |"make"| is not GNU make (|make| is always GNU make).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ocaml/opam-repository/pull/4366/files#r33881672>.
>
I don't understand what you mean.
Do you suggest using:

 1) [ "make" "uninstall" ]

 2) [ make "uninstall" ]

 3) [ "${make}" "uninstall" ]